application/extensions/LimeTopbarWidget/views/topbar.php

```diff
@@ -9,15 +9,16 @@
     <div class="container-fluid">
         <div class="row">
             <!-- Title or breadcrumb -->
-            <div class="ls-breadcrumb col-xl-4 col-xxl-3">
+            <div class="ls-breadcrumb col-12">
                 <h1 class="align-items-center d-flex">
-                <?php
-                    if ($titleBackLink !== null) {
-                        echo '<a class="h1" href="'.$titleBackLink.'">
-                        <i class="ri-arrow-left-s-line"></i></a>' ;
-                    }
-                ?>
-                <?= $leftSide ?></h1>
+                    <?php
+                        if ($titleBackLink !== null) {
+                            echo '<a class="h1" href="'.$titleBackLink.'">
```

Comment:
The text inside is the icon, right?

Reply:
Can be (for example)

`<i class="ri-arrow-left-s-line" aria-hidden="true"></i> gt('Back")`

It mean back ?

Reply:
What i mean is that there is some icon in line 17. So there is never no text for the link.

Reply:
An icon is **NOT** a text; text is for screen reader, 

see https://github.com/LimeSurvey/LimeSurvey/blob/f10805aee259d9cfba72c35e3c5e6833a714519d/README.md?plain=1#L43

Reply:
Sorry, don't understand.
This is just the same as it was before. Not sure what your comment is about.
Related to the reported bug or to an extra issue you see?

Should we add some text for screen readers? Is that what you say?

Reply:
You rewrite breadcrumb display, it's not a new issue : it still Breadcrumb is broken.

Yes : since LimeSurvey say WCAG 2.0 compliance : we NEED a real text for each icon.

Reply:
> Yes : since LimeSurvey say WCAG 2.0 compliance : we NEED a real text for each icon.

OK, shall we create a new ticket for this?